packages/govuk-frontend/src/govuk/components/skip-link/skip-link.mjs

```diff
@@ -51,20 +52,19 @@ export class SkipLink extends GOVUKFrontendComponent {
 
     // Check for link hash fragment
     if (!linkedElementId) {
-      throw new ElementError('$module.hash', {
-        componentName: 'Skip link',
-        element: this.$module,
-        expectedType: 'string'
-      })
+      throw new ElementError(
+        'Skip link: Root element (`$module`) attribute (`href`) has no URL fragment'
+      )
     }
 
     const $linkedElement = document.getElementById(linkedElementId)
 
     // Check for link target element
     if (!$linkedElement) {
-      throw new ElementError(`$module.hash target #${linkedElementId}`, {
+      throw new ElementError({
         componentName: 'Skip link',
-        element: $linkedElement
+        element: $linkedElement,
+        identifier: `Linked element #${linkedElementId}`
```

Comment:
Knowing that the default is to "skip to the main content" should we give this a tweak?

Keeping to HTML is the `id=""` format helpful?

```suggestion
        identifier: `Target content `id="${linkedElementId}"`
```